espflash/src/cli/mod.rs

```diff
@@ -716,6 +717,14 @@ pub fn erase_region(args: EraseRegionArgs, config: &Config) -> Result<()> {
         return Err(Error::StubRequired).into_diagnostic();
     }
 
+    if args.address % FLASH_SECTOR_SIZE as u32 != 0 || args.size % FLASH_SECTOR_SIZE as u32 != 0 {
+        return Err(Error::InvalidEraseRegionArgument(format!(
+            "Address ({}) and size ({}) must be multiples of {}",
+            args.address, args.size, FLASH_SECTOR_SIZE
+        ))
+        .into());
+    }
```

Comment:
I think we should have an error that takes two args (address and size), something like: 
```rust
    #[error("Invalid `address`({address})  and/or `size`({size}) argument(s)")]
    #[diagnostic(code(espflash::erase_region::invalid_argument)),
        help("`address` and `size`must be multiples of 0x1000 (4096)")]
    InvalidEraseRegionArgument {
        address: u32,
        size: u32,
    },

    #[error("Invalid erase region argument: {0}")]
    InvalidEraseRegionArgument(String),
```